docs/components/periodic-updates.rst

```diff
@@ -0,0 +1,54 @@
+DKAN Periodic Updates
+=====================
+
+The DKAN Periodic Updates module provides the user the ability to execute automated updates on existing resources in a daily, weekly or monthly basis. It also provides a UI in which you can check at the status of the updates.
```

Comment:
Documentation suggested changes will be in bold only for clarity.

...existing resources **on** a daily, weekly...